src/types/common.ts

```diff
@@ -14,6 +14,11 @@ export interface SetComponent {
   symbol: string;
 }
 
+export interface RequiredComponents {
```

Comment:
Let's call this something else since its an object, but collides with a parameter of an issuance order that is an array. Prefer interface is called `RequiredComponent` and `calculateRequiredComponentsAndUnitsAsync` returns an array of `RequiredComponents`.

Reply:
+1